index.html

```diff
@@ -148,87 +175,118 @@ <h2 class="head2">Tonic</h2>
-                <button class="see-project1">
+                <button class="see-project1 unique4">
                     <div class="see-project">See Project</div>
                 </button>
+                </div>
             </section>
         </div>
         <section>
             <div></div>
             <div></div>
         </section>
     </section>
-    </div>
-    <section class="section6">
+    <section class="section2 section2d">
+        <div class="aboutd">
         <h2 class="h">About <br>Myself</h2>
-        <p class="info11">Hello I’m a software developer! I can help you build a product , feature or website Look
+        <p class="info11 info11d">Hello I’m a software developer! I can help you build a product , feature or website Look
             through some of my work and experience! If you like what you see and have a project you need coded, don’t
             hestiate to contact me.</p>
         <p class="lc info2">LET’S CONNECT</p>
-        <footer class="foot">
+        <div class="socials">
             <div class="left">
                 <ul class="footer-items">
-                    <li class="dotless twitter"><img src='images/Vector.svg' alt="twitter icon"></li>
-                    <li class="dotless insta"><img class="iinsta" src='images/i.png'><img class="ninsta"
+                    <li class="dotless twitter status"><img src='images/Vector.svg' alt="twitter icon"></li>
+                    <li class="dotless insta status"><img class="iinsta" src='images/i.png'><img class="ninsta"
                             src='images/n.png' alt="instagram icon"></li>
-                    <li class="dotless m"><img src='images/M.svg' alt="m icon"></li>
-                    <li class="dotless gh"><img src='images/gitHub.svg' alt="github icon"></li>
-                    <li class="dotless peace"><img src='images/peace.svg' alt="peace icon"></li>
+                    <li class="dotless m status"><img src='images/M.svg' alt="m icon"></li>
+                    <li class="dotless gh status"><img src='images/gitHub.svg' alt="github icon"></li>
+                    <li class="dotless peace status"><img src='images/peace.svg' alt="peace icon"></li>
+
+                    <li class="dotless gh statusd"><img src='images/gitHub.svg' alt="github icon"></li>
+                    <li class="dotless insta statusd"><img class="iinsta" src='images/i.png'><img class="ninsta"
+                        src='images/n.png' alt="instagram icon"></li>
+                    <li class="dotless peace statusd"><img src='images/peace.svg' alt="peace icon"></li>
+                    <li class="dotless twitter statusd"><img src='images/Vector.svg' alt="twitter icon"></li>
+                    <li class="dotless m statusd"><img src='images/M.svg' alt="m icon"></li>
+                    
+                    
                 </ul>
             </div>
-        </footer>
         <button class="resume">
             <div class="text-resume">Get my resume</div>
```

Comment:
- [ ] Kindly add button interaction style for this button as shown in Figma 👍 

![image](https://user-images.githubusercontent.com/50721479/206857666-c7439e53-d90e-46aa-a681-1f9b5b1db12c.png)